### ISSUE TITLE
Sort API results on Elasticsearch highlight count (PITs per Feature)

### ISSUE THREAD

**Issue by bertspaan** [erfgoed-en-locatie/histograph#21]
Sort API results on Elasticsearch highlight count (PITs per Feature)!

**Comment by sbocconi** [comment]
Is this related to issues histograph/neo4j-plugin#9 and histograph/neo4j-plugin#10?

**Comment by bertspaan** [comment]
No, this is enhancement for tools like viewer, to show which of the API results (and what field) matched the search query: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-highlighting.html
